$NetBSD: patch-af,v 1.1 2011/01/29 21:15:11 markd Exp $

fix build with gcc4.3
curl/types.h is not used (empty) since curl-7.12.0 and removed at curl-7.21.7.

--- lib/protocol.cpp.orig	2006-11-28 20:25:04.000000000 +0000
+++ lib/protocol.cpp
@@ -8,11 +8,11 @@
 -------------------------------------------------------------------*/
 #include <stdio.h>
 #include <stdlib.h>
+#include <string.h>
 #include <string>
 #include <map>
 #include <expat.h>
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 using namespace std;
 
